### PR TITLE
Refactor: Refactor to change the modify profile process and slicing process in 3dp

### PR DESCRIPTION
**Pull request by jane-rose**

### CODE REVIEW

File: src/app/models/SvgModel.js [hunk 1]
```diff
@@ -520,14 +520,14 @@ class SvgModel extends BaseModel {
...
-
-        this.uploadImageName = res.body.uploadName;
-        this.generateModelObject3D();
-        this.generateProcessObject3D();
+        document.body.removechild(canvas);
+        const formdata = new FormData();
+        formdata.append('image', file);
+        const res = await api.uploadimage(formdata);
+
+        this.uploadimagename = res.body.uploadname;
```
Comment (xiekun1992) [inline src/app/models/SvgModel.js:528]: this renaming will cause some potential problems

File: src/app/models/ModelGroup.js [hunk 2]
```diff
@@ -2181,7 +2174,7 @@ class ModelGroup extends EventEmitter {
         let maxHeight = 0.1;
         const maxBoundingBoxHeight = this._bbox?.max.z;
         this.models.forEach(modelItem => {
-            if (modelItem.headType === HEAD_PRINTING && modelItem.type !== 'primeTower' && !modelItem.supportTag) {
+            if (modelItem.headType === HEAD_PRINTING && !modelItem.supportTag) {
```
Comment (xiekun1992) [inline src/app/models/ModelGroup.js:2177]: `!modelItem.supportTag` will always be `true`

**Review by xiekun1992 — CHANGES_REQUESTED** [review]

File: src/app/models/SvgModel.js (hunk 1 above)
Comment (jane-rose) [reply src/app/models/SvgModel.js:528]: this will be fixed on other pr

**Review by MaShizhen — APPROVED** [review]

**Comment by jane-rose** [comment]
https://snapmaker2.atlassian.net/browse/LU-2370

**Pull request merged**